Pet-Clinic-Frontend/src/App.tsx

```diff
@@ -1,16 +1,15 @@
-
-
-import './App.css'
+import './App.css';
+import clinicImage from './images/petclinic.png'; 
 
 function App() {
-
-
   return (
     <>
-    <h1>Main page</h1>
-  
+     <img src={clinicImage} alt="Clinic" className="clinic-image" /> 
+      <h1 className="h1">Pet Clinic</h1>
+      <h2 className="h2">Welcome to our fabulous clinic</h2>
+      
```

Comment:
try to put it as a separate page into the pages folder, than call it here (like <LandingPage />)

Reply:
Done :))